src/CLI/actioner/sonic-cli-mac.py

```diff
@@ -69,45 +58,45 @@ def mac_fill_count(mac_entries):
 
 def fill_mac_info(mac_entry):
     mac_entry_table = {'Vlan':mac_entry['vlan'], 
-                        'mac-address':mac_entry['mac_address'],
-                        'entry-type': mac_entry['state']['entry_type'],
-                        'port': mac_entry['interface']
-                                ['interface_ref']['state']['interface']
+                        'mac-address':mac_entry['mac-address'],
+                        'entry-type': mac_entry['state']['entry-type'],
+                        'port': mac_entry['interface']['interface-ref']['state']['interface']
     }
     return mac_entry_table
 
+def invoke(func, args):
+    body = None
+    aa = cc.ApiClient()
 
-def run(func, args):
-
-    c = openconfig_network_instance_client.Configuration()
-    c.verify_ssl = False
-    aa = openconfig_network_instance_client.OpenconfigNetworkInstanceApi(api_client=openconfig_network_instance_client.ApiClient(configuration=c))
-
-    # create a body block
-    keypath, body = generate_body(func, args)
+    if func == 'get_openconfig_network_instance_network_instances_network_instance_fdb_mac_table_entries':
+        keypath = cc.Path('/restconf/data/openconfig-network-instance:network-instances/network-instance={name}/fdb/mac-table/entries', name='default')
+        return aa.get(keypath)
 
+def run(func, args):
     try:
-        if body is not None:
-            api_response = getattr(aa, func.__name__)(*keypath, body=body)
-
-        else:
-            api_response = getattr(aa,func.__name__)(*keypath)
-
-        if api_response is None:
-            print ("Success")
-        else:
-            response = api_response.to_dict()
+        api_response = invoke(func,args)
+
+        if api_response.ok():
+            response = api_response.content
+            if response is None:
+                print "Success"
+                return
+            else:
+                if 'openconfig-network-instance:entries' in response:
+                    mac_entries = response['openconfig-network-instance:entries']['entry']
+                else:
```

Comment:
Let's remove this.

Reply:
If system doesn't have any mac address than this will give graceful exit with print statement or else it will simply exit with no message. Let me know if still need to remove this.

Reply:
done